packages/amplify-cli/src/index.ts

```diff
@@ -18,6 +20,13 @@ EventEmitter.defaultMaxListeners = 1000;
 // entry from commandline
 export async function run(): Promise<number> {
   try {
+    // Checks for available update on each use
+    const notifier = updateNotifier({
+      pkg,
+      updateCheckInterval: 0,
+    });
+    notifier.notify({ defer: false, isGlobal: true });
```

Comment:
What do you think if the update notification would appear after the logo and such but after execution in general, otherwise it could be missed easily since we output a lot of information. What is the practice in other major tools that using this package?

Reply:
From looking at the behavior other CLI's (npm, angular, yeoman) and considering the amount of information we output, the notification should appear after execution. Making this change